warehouse-management-service/pkg/database/postgres/shelf_block.go

```diff
@@ -0,0 +1,200 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	wms "warehouse-management-service"
+)
+
+type shelfBlockQueries interface {
+	createShelfBlockTx(ctx context.Context, tx *sql.Tx, block wms.ShelfBlock) error
+	getShelfBlockByIdTx(ctx context.Context, tx *sql.Tx, id string) (wms.ShelfBlock, error)
+	updateShelfBlockTx(ctx context.Context, tx *sql.Tx, block wms.ShelfBlock) error
+	deleteShelfBlockTx(ctx context.Context, tx *sql.Tx, id string) error
+	warehouseExistsTx(ctx context.Context, tx *sql.Tx, id string) (bool, error)
+}
+
```

Comment:
Should we add the go-mockgen command for this?

Reply:
Added, missed this.